src/components/ProductList/ProductList.js

```diff
@@ -37,12 +43,12 @@ const ProductList = ({ selectedFilter, getProducts, productsList }) => {
       ) : (
         <></>
       )}
-      {productsList.loading ? (
+      {list.loading ? (
```

Comment:
I didn't test it yet, but I think that this should be just `loading`. This is also a note for me, I will check this.

Reply:
You are right, just "loading"

Reply:
Done